.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Unit tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
We run CI on PRs pointing to any branch not only master and it is something we are following in all repos across edx. For instance you can have a look at this [CI FILE](https://github.com/edx/edx-submissions/blob/master/.github/workflows/ci.yml)